ftdc/custom_format_test.go

```diff
@@ -266,3 +266,30 @@ func TestNestedReflectionParity(t *testing.T) {
 	test.That(t, values, test.ShouldResemble,
 		[]float32{1, 3, 4, 6, 7, 9, 11, 12, 13, 17})
 }
+
+type nestsAny struct {
+	Number float32
+	Struct any
+}
+
+func TestNestedAny(t *testing.T) {
+	logger := logging.NewTestLogger(t)
+
+	stat := nestsAny{10, struct{ X int }{5}}
+	fields, err := getFieldsForStruct(reflect.ValueOf(stat))
+	logger.Info("Fields:", fields, "Err:", err)
+	// ["Number", "Struct.X"]
```

Comment:
I guess I can add the assertions now! oops

Reply:
Yeah assertions here would be great.